src/tools/nngcat/nngcat.c

```diff
@@ -705,6 +709,36 @@ sendrecv(nng_socket sock)
 	}
 }
 
+void
+pipeloop(nng_socket sock)
+{
+	if (pipefile == NULL) {
+		fatal("No pipe to send (specify with --pipe)");
+	}
+	if (delay > 0) {
+		nng_msleep(delay);
+	}
+
+	int          rv;
+	nng_msg *    msg;
+  char * line = NULL;
```

Comment:
Minor indenting error.  We use indents by tab, where the tab is 8 cells wide.  Please see our .clang-format.